### PR TITLE
BZ1973326: Add 4-8 RN for multipath

### PR DESCRIPTION
**Pull request by bobfuru**
Adds 4.8 Release Note for enabling multipathing at installation.

Relates to https://github.com/openshift/openshift-docs/pull/34462, which will be merged post-GA.

https://bugzilla.redhat.com/show_bug.cgi?id=1973326
Relates to content added as part of https://github.com/openshift/openshift-docs/pull/34462

Preview: https://deploy-preview-34798--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-8-release-notes?utm_source=github&utm_campaign=bot_dp#ocp-4-8-rhcos-

### CODE REVIEW

**Comment by bobfuru** [comment]
@jlebon and @miabbott PTAL

**Review by jlebon — APPROVED** [review]
Should we link to the other doc you're working on in that other PR detailing the steps or is that not usually the format for release notes?

File: release_notes/ocp-4-8-release-notes.adoc
Comment (miabbott) [inline release_notes/ocp-4-8-release-notes.adoc:87]: Is there any platform restriction on multipath support? Or do we actually recommend this for all platforms?

Comment (bobfuru) [reply release_notes/ocp-4-8-release-notes.adoc]: > Is there any platform restriction on multipath support? > Or do we actually recommend this for all platforms?

Hi @miabbott - My understanding is that it would be recommended for bare metal, so I've clarified that in the wording. Is this correct, @jlebon?

Also, I've clarified the wording so it doesn't sound like _all_ clusters should have multipath enabled. The main point is that we recommend multipathing at install over configuring it as a day-2 operation, not as a general principle.

**Comment by bobfuru** [comment]
> Should we link to the other doc you're working on in that other PR detailing the steps or is that not usually the format for release notes?

@jlebon - We'll add the link once that related PR (https://github.com/openshift/openshift-docs/pull/34462) is ready and merged. I've added the link to the file and commented it out but will issue an updated PR as soon as merge freeze is lifted.

**Review by jlebon — APPROVED** [review]
Looks great!

File: release_notes/ocp-4-8-release-notes.adoc
Comment (kalexand-rh) [inline release_notes/ocp-4-8-release-notes.adoc:84]:
```suggestion
[id="ocp-4-8-rhcos-multipath-install"]
```

File: release_notes/ocp-4-8-release-notes.adoc
Comment (kalexand-rh) [inline release_notes/ocp-4-8-release-notes.adoc:87]:
```suggestion
Enabling multipathing during bare metal installation is now supported for nodes provisioned in {product-title} 4.8 or higher. You can enable multipathing by appending kernel arguments to the `coreos-installer install` command so that the installed system itself uses multipath beginning from the first boot. While post-installation support is still available by activating multipathing through the machine config, enabling multipathing during installation is recommended for nodes provisioned starting in {product-title} 4.8.
```

> ! via preposition
Use this term, which means "by way of", to refer to routing data, such as in "Messages are routed via SMTP". In other situations, try to replace "via" with one of the following synonyms: "across", "along", "by", "from", "on", or "through".

**Review by kalexand-rh — COMMENTED** [review]
I have a couple of picks, but this looks good!

**Review by jeana-redhat — APPROVED** [review]
Approved for merge pending Kathryn's changes :)

File: release_notes/ocp-4-8-release-notes.adoc
Comment (bobfuru) [reply release_notes/ocp-4-8-release-notes.adoc:87]: Thanks, @kalexand-rh.

- I contemplated "uses" rather than "will use" in this particular case because it seemed a bit of a sequential process: append kargs now so that it will be used at first boot. I was probably overthinking it, and "uses" has the benefit of sounding fine and also conforming to ISG. Changed.
- I also contemplated replacing "via" but nothing sounded more appropriate in this case. IMO, "activating multipathing _through_ the machine config" makes it sound like it's actually traveling through the MC, which seems odd. In this occurrence, I take it to actually mean "by way of", so it seems "via" is the most appropriate. Unchanged.

File: release_notes/ocp-4-8-release-notes.adoc
Comment (bobfuru) [reply release_notes/ocp-4-8-release-notes.adoc]: Good catch, thanks!

**Pull request merged**
